server.ts

```diff
@@ -22,12 +22,35 @@ app.use(cors())
 app.use(bodyParser.json())
 
 new RateLimiter(app, [GLOBAL_RL]);
-new RateLimiter(app, evmchains);
+
+new RateLimiter(app, [
+    ...evmchains,
+    ...erc20tokens
+]);
 
 const captcha = new VerifyCaptcha(app, process.env.CAPTCHA_SECRET!);
 
 let evms: any = {};
 
+const getChainByID = (chains: any, id: string): any => {
+    let reply;
+    chains.forEach((chain: any) => {
+        if(chain.ID == id) {
+            reply = chain;
+        }
+    });
+    return reply;
+}
+
+const populateConfig = (child: any, parent: any) => {
```

Comment:
I see `any` as a bunch of types. Do we not know the actual type? VSCode will often tell you if you hover your mouse over the variable for function.

Reply:
I have updated the code with new types wherever possible. Thanks.